questions/includes/fedora/coding.yml

```diff
@@ -8,8 +8,24 @@ tree:
 
     - title: C
       subtitle: close to the metal
-      href: http://lkml.org
+      href: https://lkml.org
+
+    - title: Haskell
+      subtitle: a nondynamic language for rapid development
+      segue1: Yay, types!
+      segue2: How about...
+      href: https://fedoraproject.org/wiki/Haskell_SIG
```

Comment:
So, I'd move the SIG link into another child node in ``includes/fedora/coding/haskell.yml``.